### PR TITLE
docs: update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by theyokohamalife**
# Resolves Issue #40 

1. Removes links to resources we no longer use
2. Updates the Slack channel information
3. Updates the technologies we use

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,43 +1,38 @@
...
+1. Fork the repo and build the source by following the [README](./README.md)
+2. Find a good issue by checking out the [Issues Section](https://github.com/ourjapanlife/findadoc-web/issues), and look for something tagged "Good First Issue"
 3. Submit a PR and work with the team to merge it
    🙌🏻
+4. Give us a shout if you need any help on the `#frontend-team` channel on Slack
 
 👉️ Translators 🌐
 
-1. [Visit the Localization Subproject](https://github.com/ourjapanlife/findadoc-localization) for further instructions
+1. Join the `#localization` channel on Slack.
```
Comment (ann-kilzer) [inline CONTRIBUTING.md:17]: We don't have translation work at this time, but will soon

**Review by ann-kilzer — APPROVED** [review]
LGTM

Let's check that Netlify build... it's definitely unrelated but still strange

**Comment by theyokohamalife** [comment]
> LGTM
> 
> Let's check that Netlify build... it's definitely unrelated but still strange

@ann-kilzer It looks like yarn is failing to download during the deployment process.

Screen Shot 2022-07-27 at 3 50 16 PM

**Comment by theyokohamalife** [comment]
@ermish Is there anything you'd like added or changed?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,43 +1,38 @@
...
 
-## What if I get stuck?
-
-- Developers: Check out the [TROUBLESHOOTING Guide](https://github.com/ourjapanlife/findadoc-frontend/wiki/Troubleshooting-%F0%9F%98%AD-%F0%9F%98%AD-%F0%9F%98%AD-%F0%9F%91%89%EF%B8%8F-%F0%9F%98%8A-%F0%9F%98%8A--%F0%9F%98%8A) or ask in Slack
-- Translators: Ask in slack in the #localization channel
-- Everyone: Ask in Slack
-
 ## Helpful Info
 
 ### Related Technologies
```
Comment (ermish) [inline CONTRIBUTING.md:26]: this section probably fits the readme better, so we could take it out here.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,43 +1,38 @@
...
+1. Join the `#ui-ux-design` and `#frontend-team` channels on Slack
    🎨
 
-## What if I get stuck?
-
-- Developers: Check out the [TROUBLESHOOTING Guide](https://github.com/ourjapanlife/findadoc-frontend/wiki/Troubleshooting-%F0%9F%98%AD-%F0%9F%98%AD-%F0%9F%98%AD-%F0%9F%91%89%EF%B8%8F-%F0%9F%98%8A-%F0%9F%98%8A--%F0%9F%98%8A) or ask in Slack
-- Translators: Ask in slack in the #localization channel
-- Everyone: Ask in Slack
-
 ## Helpful Info
```
Comment (ermish) [inline CONTRIBUTING.md:24]: should remove since empty for now, unless we're planning to add content here soon

**Review by ermish — CHANGES_REQUESTED** [review]
reviewed!

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,43 +1,38 @@
-Welcome to Findadoc Japan! We're grateful for our volunteers 😊
+Welcome to Find a Doc, Japan! We're grateful for our volunteers 😊
 
 # Where should I start?
 
-👉️ Join the [Slack](https://join.slack.com/t/find-a-doc/shared_invite/zt-s4744a6o-MGaGHzLN5wB9aXeha3vdsQ) 💬
+👉️ Join us on [Slack](https://join.slack.com/t/find-a-doc/shared_invite/zt-s4744a6o-MGaGHzLN5wB9aXeha3vdsQ) 💬
 
 👉️ Developers 💻️
```
Comment (ermish) [inline CONTRIBUTING.md:7]: might make these sections H3/H4 headers

**Comment by ermish** [comment]
rebasing fixed the build ✨ 😺

**Comment by theyokohamalife** [comment]
> rebasing fixed the build ✨ 😺

@ermish  Hmm... rebasing it shouldn't show all of the file changes though. Only the README and CONTRIBUTING should be changed. 🤔

**Comment by ann-kilzer** [comment]
lol 
> +11,782 −25,323

make sure you have the latest main before rebasing???

**Review by ann-kilzer — CHANGES_REQUESTED** [review]
please fix the branch and then ping me and I'll approve asap 😊

**Review by ann-kilzer — APPROVED** [review]

**Comment by theyokohamalife** [comment]
@ermish I fixed the rebase. Please review when you have a chance

**Review by ermish — APPROVED** [review]

**Comment by ermish** [comment]
LGTM :shipit:

**Pull request merged**